### PR TITLE
Update background.js

### PR DESCRIPTION
**Pull request by koops76**

### CODE REVIEW

File: chromium/background.js [hunk 1]
```diff
@@ -679,36 +679,35 @@ chrome.runtime.onMessage.addListener(function(message, sender, sendResponse){
 // Send a message to the embedded webextension bootstrap.js to get settings to import
 chrome.runtime.sendMessage("import-legacy-data", import_settings);
 
+function setStorage(values) {
+  return new Promise(resolve => {
+    storage.set(values, resolve);
+  }
```
Comment (Hainish) [inline chromium/background.js]: Add closing parenthesis here.  Also, I'd rather have this in the `import_settings` function.  Just call await to the promise.  But consolidating the four separate calls is :+1:.

Comment (koops76) [inline chromium/background.js]: @Hainish Thanks for pointing it out!

**Comment by Hainish** [comment]
Great now

> Also, I'd rather have this in the import_settings function. Just call await to the promise

**Comment by koops76** [comment]
@Hainish Done.

**Comment by Hainish** [comment]
Thanks!

**Pull request merged**